data/Drug_target_interaction/meta.yaml

```diff
@@ -0,0 +1,45 @@
+---
+name: Drug-Target Interaction
+description: |-
+    The activity of a small-molecule drug is measured by its binding affinity with the target protein.
+            Given a new target protein, the very first step is to screen a set of potential compounds to find their activity.
+            Traditional method to gauge the affinities are through high-throughput screening wet-lab experiments.
+            However, they are very expensive and are thus restricted by their abilities to search over a large set of candidates
+            Drug-target interaction prediction task aims to predict the interaction activity score in silico given only the accessible compound structural information and protein amino acid sequence.
+targets:
+    - id: binding
+      description: small-molecule protein interaction.
+      units: Kd
```

Comment:
what is this in SI units? (I'm really ignorant, sorry)

Reply:
Mole?